server/matchmaker/search.py

```diff
@@ -40,9 +43,29 @@ def __init__(self, player, start_time=None, rating_prop='ladder_rating'):
             0: 0.8
         }
 
+        self.adjusted_rating = self.adjust_rating()
```

Comment:
Rather than assigning it here you'll probably want to make `adjusted_rating` a computed proper like `rating`:

```python
@property
def adjusted_rating(self): ...
```

And inline the definition of `adjust_rating`.